site/src/Controller/ProjectController.php

```diff
@@ -162,13 +164,17 @@ public function newAction(
         // save project and redirect to project page
         if ($form->isSubmitted() && $form->isValid()) {
             // populate Project with data from "special"/not mapped form fields
-            $this->populateProjectFromForm(
-                $p,
-                $form,
-                $githubApiService,
-                $organizationRepository,
-                $userManager
-            );
+            try {
+                $this->populateProjectFromForm(
+                    $p,
+                    $form,
+                    $githubApiService,
+                    $organizationRepository,
+                    $userManager
+                );
+            } catch (ErrorException $e) {
```

Comment:
Can we find a more focused Exception to catch, or event give the user bits of the error message we got from GitHub?